collage update 1

jht301 opened the pull request:
I'm working on the slideshow type thing and I'm not sure where i messed
up. I apologize that my code is a little sloppy. basically i created the
array images[] put the pictures in it and i can't get them to return.

jht301 committed:
collage update 1

I'm working on the slideshow type thing and I'm not sure where i messed
up. I apologize that my code is a little sloppy. basically i created the
array images[] put the pictures in it and i can't get them to return.

LadyK commented:
What does the console tell you? Don't forget about that. It's key! As well as console.log (P5.js) + print statements(Processing)

jht301 commented:
i keep getting the error TypeError: a is undefined    p5.min.js:7:14096  
have you ever seen that before?

LadyK commented:
Make sure you are calling and loading files from the correct place. Use the absolute URL for P5:

<meta charset=utf8>

<script src=http://p5js.org/js/p5.min.js></script>

What does your index.html file say?

jht301 committed:
update

still not sure what's wrong. i tried looking up the problem but couldnt
find anything

jht301 commented:
i kept everything in the html file the same. it says what it normally does. i just pushed another update can you see the html file?

LadyK commented:
What is the issue you are getting? I don't have any issues, but I do see issues in your code.

do a console.log(" var i is =  " + i);

Also, watch this short series regarding debugging. I keep mentioning this methods in class. They are worth their weight in gold:

https://vimeo.com/channels/debugging

jht301 committed:
update

jht301 commented:
pushed it

jht301 committed:
project in processing

i finished the project in processing. it's very slow. but it works.
